frontend/src/pages/QADashboard/index.js

```diff
@@ -1,7 +1,11 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
+import { Grid, Alert } from '@trussworks/react-uswds';
+import QAOverview from '../../widgets/QualityAssuranceDashboardOverview';
 
 export default function QADashboard() {
+  const [error] = useState(null);
```

Comment:
are we likely to set/display an error in here? or are we setting an error at the point of fetching data for the page? (the parent page)

Reply:
whoops my bad this is the top level, I got confused